coso/.htaccess

```diff
@@ -0,0 +1,40 @@
+# # /coso/
+#
+# http://w3id.org/coso/ html request redirects to 
+# https://gdb.acg.maine.edu:7201/resource?uri=http:%2F%2Fw3id.org%2Fcoso%2Fv1%2Fcontaminoso%23 for html request
+# to https://raw.githubusercontent.com/SAWGraph/contaminoso/refs/heads/main/v1/contaminoso_core.ttl for base request
+#
+# http://w3id.org/coso/ ttl request redirects to
```

Comment:
- You should use `https` even in docs.
- That maine.edu target is not in the rules below.
- Last line trails off.